source/SUBMIT.rst

```diff
@@ -126,40 +126,38 @@ Example preprint templates using ``repo2data`` for caching data on NeuroLibre se
    * - Nilearn
      - `neurolibre/repo2data-nilearn <https://github.com/neurolibre/repo2data-caching>`_
    * - OSF
-     - `neurolibre/repo2data-osf <https://github.com/ltetrel/neurolibre-osf-test>`_
+     - `neurolibre/repo2data-osf <https://github.com/neurolibre/neurolibre-osf-test>`_
 
 
 
-.. warning:: RoboNeuro may fail downloading relatively large datasets (**exceeding 5GB**) as the book build process times out in 30 minutes.
+.. warning:: 
+  RoboNeuro may fail downloading relatively large datasets (**exceeding 5GB**) as the book build process times out in 60 minutes.
+  If you have data that exceed 5GB, please create an issue in your github repository so a Neurolibre admin can check it.
 
 .. topic:: Help RoboNeuro find your data during book build
 
-  RoboNeuro downloads your data to a folder named ``data``, which is created at the base of your repository.
+  `Repo2Data <https://github.com/SIMEXP/Repo2Data>`_ downloads your data to a folder named ``data/PROJECT_NAME``, which is created at the base of your repository.
+  ``PROJECT_NAME`` is `the field <https://github.com/neurolibre/neurolibre-osf-test/blob/1edcdb82a5c3f52130572f5cd6e4f1386ae3d8e4/binder/data_requirement.json#L3>`_ that you configured in your ``data_requirement.json``.
 
   - A code cell in a ``content/my_notebook.ipynb`` would access data by:
 
     .. code-block:: python
 
       import nibabel as nib
       import os
-      img = nib.load(os.path.join('..','data','my_brain.nii.gz'))
+      img = nib.load(os.path.join('..', 'data', 'PROJECT_NAME', 'my_brain.nii.gz'))
 
   - A code cell in a ``content/01/my_01_notebook.ipynb`` would access data by:
 
     .. code-block:: python
 
       import nibabel as nib
-      img = nib.load(os.path.join('..','..',data','my_brain.nii.gz')) # In this case, 2 upper directories
+      img = nib.load(os.path.join('..', '..', 'data', 'PROJECT_NAME', 'my_brain.nii.gz')) # In this case, 2 upper directories
 
   If the data directories in your code cells are not following this convention, RoboNeuro will fail to re-execute your notebooks and interrupt the book build.
 
-.. note:: When running your notebooks locally, we suggest that you to load data from a ``data`` folder located at the base of your repository content.
-          This way your data loading convention will match that of RoboNeuro's and you won't have remember changing directories each time you request a book preview service.
-          
-          **Please make sure that you ignore this folder and all of its contents from Git history by adding the following in the** ``.gitignore`` **file:**
-            .. code-block:: text
-
-              data/
+.. note:: We suggest you to follow `this section <#testing-book-locally>`_,
```

Comment:
Reword: We suggest testing `repo2data` locally before you request a RoboNeuro preview service. Instructions are available `here <#testing-book-locally>`_. Matching your data loading convention with that of RoboNeuro will increase your chances of having a successful NeuroLibre preprint build.

Reply:
nice